tests/plugins/embed/manual/embed.md

```diff
@@ -1,3 +1,4 @@
+@bender-tags: feature, generic
```

Comment:
Please remove broken links:
`https://www.google.dk/maps/place/Zygmunta+S%C5%82omi%C5%84skiego+15,+Warszawa` and
`http://tellto.tumblr.com/post/112160646328`
from the list.